### PR TITLE
ORLP-170. Bugs have been fixed.

### PR DESCRIPTION
**Pull request by b0hdan**

### CODE REVIEW

File: src/app/page/cabinet/userDecks/user.decks.component.html [hunk 1]
```diff
@@ -95,9 +95,13 @@ <h4 class="modal-title">{{dialogName}}</h4>
           </table>
         </div>
         <div class="modal-footer">
-          <button *ngIf="name !== '' && description !== '' && categoryId != -1" class="btn btn-primary btn-pointer"
+          <button *ngIf="name && description && categoryId != -1" class="btn btn-primary btn-pointer"
```
Comment (antonbakalets) [inline src/app/page/cabinet/userDecks/user.decks.component.html]: -1 is still here...

File: src/app/page/cabinet/userDecks/user.decks.component.html [hunk 2]
```diff
@@ -95,9 +95,13 @@ <h4 class="modal-title">{{dialogName}}</h4>
...
                   data-dismiss="modal" aria-label="Close" (click)="createDeck()">{{dialogButtonName}}
           </button>
+          <button *ngIf="name === '' || description === '' || categoryId == -1"
```
Comment (antonbakalets) [inline src/app/page/cabinet/userDecks/user.decks.component.html]: The condition hasn't changed here.

File: src/app/page/cabinet/userDecks/user.decks.component.html (hunk 1 above)
Comment (b0hdan) [reply src/app/page/cabinet/userDecks/user.decks.component.html]: Sorry, I have no idea how to fix that. I have tried a lot of different ways but noone works.
Of course, I can assign that variable undefined or null but in any case it'll be interpreted as a string.

File: src/app/page/cabinet/userDecks/user.decks.component.html (hunk 2 above)
Comment (b0hdan) [reply src/app/page/cabinet/userDecks/user.decks.component.html]: Yes, I'll fix it in the moment.

**Pull request merged**